app/processors/join.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Joins values using configured value or linebreak
+class Join < Blacklight::Rendering::AbstractStep
+  include ActionView::Helpers::TextHelper
+
+  def render
+    # rubocop:disable Rails/OutputSafety
+    joiner = config.join_with || '<br>'.html_safe
```

Comment:
I think this can be done now using `separator_options` (used as arguments to https://apidock.com/rails/Array/to_sentence)? It's not quite as clean, since we'll probably have to set all 3 options, but I think it should work.

Reply:
@cbeer Yes, I considered that, but then we'd have to set separator_options on every field.  This seems nicer.

Reply:
:shrug:. I guess we can revisit it if we ever want to use the upstream code too.